packages/governance/src/closingRule.js

```diff
@@ -10,7 +10,7 @@ import { Far } from '@endo/marshal';
 /** @type {CloseVoting} */
 export const scheduleClose = (closingRule, closeVoting) => {
   const { timer, deadline } = closingRule;
-  E(timer).setWakeup(
+  return E(timer).setWakeup(
```

Comment:
Was the need for the `return` here a genuine bug noticed by these linting rules, or was the `return` just needed to satisfy the rules. It is certainly an observable semantic change! (whereas `void`, IIUC, would satisfy the same linting rule but not change the semantics)

Reply:
To satisfy the rule either `return` or `void` was necessary. I went with `return` because I figured allowing the caller of `scheduleClose` to get rejections would be preferred. Granted it's a semantic change.

I'll try to make sure that any semantic changes are instead only _suggested_ in this PR though the use of a eslint-disable comment.